SecurityPkg/Include/Guid/ExcludedFvHob.h

```diff
@@ -0,0 +1,19 @@
+/** @file

```

Comment:
Can you change the email to Sean Brogan's other email that isn't associated with uefibot?